pymc/tests/test_mixture.py

```diff
@@ -657,6 +658,28 @@ def test_iterable_single_component_warning(self):
         with pytest.warns(UserWarning, match="Single component will be treated as a mixture"):
             Mixture.dist(w=[0.5, 0.5], comp_dists=[Normal.dist(size=2)])
 
+    @pytest.mark.parametrize(
+        "weights, comp_dists, size, expected",
+        [
+            (
+                np.array([1, 0]),
+                [Normal.dist(-2, 5), Normal.dist(2, 5)],
+                None,
+                -2,
+            ),
+            (
+                np.array([0.5, 0.5]),
+                [Normal.dist(-2, 5), Normal.dist(2, 5)],
+                None,
+                0,
+            ),
+        ],
+    )
+    def test_mixture_moments(self, weights, comp_dists, size, expected):
+        with Model() as model:
+            Mixture("m", weights, comp_dists, size)
```

Comment:
I think this is why they're failing
```suggestion
            Mixture("m", weights, comp_dists, size=size)
```